src/Execution/Arguments/ArgPartitioner.php

```diff
@@ -187,6 +188,10 @@ public static function methodReturnsRelation(
             return false;
         }
 
+        if (! class_exists($returnType->getName())) {
+            throw new Exception('Class '.$returnType->getName().' does not exist, did you forget to alias the Eloquent relation class?');
```

Comment:
I think we should use a more particular exception type, maybe `DefinitionException`?

Reply:
@spawnia fine by me - I was going to use `DefinitionException` but thought it might be reserved for exceptions related directly to GraphQL language.

Reply:
The current Docblock said so, good call. It is however meant to be used a bit more broadly than that, cases where the PHP code does not match the schema are also covered by it.